minor cleanups (#3073)
uyuni-project/uyuni-docs: modules/installation-and-upgrade/pages/container-deployment/suma/server-deployment-suma.adoc

// This sections needs to be run by devs. How would a customer use custom storage? How do we set this up with mgradm?

{productname} {productnumber} defines the required persistent storage volumes by default.
These are created during installation by the [command]``mgradm`` tool if they do not already exist.

These volumes are created in [path]``/var/lib/containers/storage/volumes/``, where [systemitem]``Podman`` stores its volumes by default.

.Recommendations
[NOTE]
====
You can take advantage of this simplicity of storage by mounting an external storage device on this directory.

> Because this is where the {postgresql} database, binary packages for repositories, caches, operating system images, autoinstallation distributions, and configuration files will all be stored, there are three recommendations:

Reviewer: I would word this section a little differently:

"You can leverage the simplicity of storage by mounting an external storage device to this directory. Since it will store the {postgresql} database, binary packages for repositories, caches, operating system images, autoinstallation distributions, and configuration files, we have three recommendations:"

Author: thanks, I hope I got it right (using the github UI).

> Because this is where the {postgresql} database, binary packages for repositories, caches, operating system images, autoinstallation distributions, and configuration files will all be stored, there are three recommendations:

Author: You can leverage the simplicity of storage by mounting an external storage device to this directory.
Since it will store the {postgresql} database, binary packages for repositories, caches, operating system images, autoinstallation distributions, and configuration files, we have three recommendations:

> You can take advantage of this simplicity of storage by mounting an external storage device on this directory.
> Because this is where the {postgresql} database, binary packages for repositories, caches, operating system images, autoinstallation distributions, and configuration files will all be stored, there are three recommendations:

Author: ```suggestion
You can leverage the simplicity of storage by mounting an external storage device to this directory.
Since it will store the {postgresql} database, binary packages for repositories, caches, operating system images, autoinstallation distributions, and configuration files, we have three recommendations:
```
